build/dockerfiles/dev.Dockerfile

```diff
@@ -6,81 +6,49 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-# https://quay.io/eclipse/che-machine-exec#^7\.
-FROM quay.io/eclipse/che-machine-exec:7.56.0 as machine-exec
+FROM quay.io/devfile/universal-developer-image:latest
```

Comment:
`universal-developer-image` has a lot of tools, but as result it has a huge image size
is it really required to have **all** those tools from the `universal-developer-image` for the `che-code` development?
can we just add required tools to the dev image?

just for my understanding, as you are working on it and you know the problem better then me...

Reply:
I had some issues with installing podmn in the current `che-dev` image.
So, It was much easier to take the universal developer image and add missing libraries, required for building vscode.